internal/slicer/slicer_test.go

```diff
@@ -857,24 +892,28 @@ func runSlicerTests(c *C, tests []slicerTest) {
 
 		archives := map[string]archive.Archive{}
 		for name, setupArchive := range release.Archives {
-			archive := &testArchive{
-				options: archive.Options{
+			archive := &testutil.TestArchive{
```

Comment:
You did not add the files to Git for `testutil.TestArchive` it seems.

Reply:
:facepalm: My bad, added in 1ccb882.